### PR TITLE
Bowen

### PR DESCRIPTION
**Pull request by minceb**

### CODE REVIEW

File: SIHRDmodel with variability.Rmd [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+                  Total.Recovered = Total.Recovered,
+                  Total.Dead = Total.Dead)
+  
+  # Watch the disease until it ends (daily new case = 0) or at max 200 days
+  while(NewInfected * S > 0 & curr.day < 200){
+    curr.day = curr.day + 1
+    
+    # Number of individuals recovered from the infected in hospital (previous day)
+    Leaving.Hospital = rbinom(1, Total.Hospital, psi)
+    Recovered.Hospital.today <- rbinom(1, Leaving.Hospital, theta)
```
Comment (LinhTangTD) [inline SIHRDmodel with variability.Rmd:63]: Can you change <- to = just for consistency? Sorry for being nit-pick but it is also easier to detect on Github lol

File: SIHRDmodel with variability.Rmd [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+                  Total.In.Hospital = Total.Hospital,
+                  # Total.NonHospital = Total.NonHospital,
+                  Infected = Total.Infected,
+                  # Total.Infective = Total.Infective,
+                  # Recovered.In.Hospital = Hospital.Recovered,
+                  # Dead.In.Hospital = Hospital.Dead,
+                  Total.Recovered = Total.Recovered,
+                  Total.Dead = Total.Dead)
+  
+  # Watch the disease until it ends (daily new case = 0) or at max 200 days
```
Comment (LinhTangTD) [inline SIHRDmodel with variability.Rmd:55]: I'm not sure how to define the "end" of the disease, should we do daily new case = 0 or infected = 0?

File: SIHRDmodel with variability.Rmd [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+}
+colnames(result_table) = c("Population", "TransmissionRate", "HospitalizationRate", "HospitalizedRecoveryRate", "Non-HospitalizedRecoveryRate", "HospitalizationTime", "InfectionTime", "LastDay", "Status", "TotalDeath", "Total Infected", "Max New Case", "PeakDayNewCase", "MaxInHospital", "PeakDayHospital")
+result_table[1:7] = as.data.frame(sapply(result_table[1:7], as.numeric))
+result_table[1:7] = format(result_table[1:7], scientific = FALSE)
+result_table[c(1:7,9)] = as.data.frame(sapply(result_table[c(1:7,9)], as.factor))
+result_table[c(8,10:15)] = as.data.frame(sapply(result_table[c(8,10:15)], as.numeric))
+```
+
+###Factorial Design of 5 factors with 288 levels
+
```
Comment (LinhTangTD) [inline SIHRDmodel with variability.Rmd:214]: Is this a duplicate by mistake or is there anything changed in the previous experiment? (line 150-288)

**Review by LinhTangTD — CHANGES_REQUESTED** [review]
Just add some notes, I think generally the code looks awesome! Thank you!

**Review by britneyhe — APPROVED** [review]

**Pull request merged**